### PR TITLE
Drop a few dashboard-related changes

### LINKED ISSUE #1331: Starred repo names not displayed in activity feed on Firefox
Appearance with RGH enabled on Firefox 60.0.1 (64-bit):

image

Appearance without RGH enabled: 

image

### PR DESCRIPTION
**Pull request by bfred-it**
GitHub changed the dashboard style. Fixes #1331

# Native

# Current PR

rgh

### CODE REVIEW

**Comment by yakov116** [comment]
`The dashboard organization switcher is moved to the right column.` Readme Update

**Comment by busches** [comment]
This will impact GHE so I'd prefer this to be backwards compatible for
those users for a bit longer.

File: source/content.js [hunk 1]
```diff
@@ -73,7 +72,6 @@ import addScopedSearchOnUserProfile from './features/add-scoped-search-on-user-p
 import monospaceTextareas from './features/monospace-textareas';
 import improveShortcutHelp from './features/improve-shortcut-help';
 import displayIssueSuggestions from './features/display-issue-suggestions';
-import addPullRequestHotkey from './features/add-pull-request-hotkey';
```
Comment (busches) [inline source/content.js]: Did you mean to remove this?

Comment (bfred-it) [reply source/content.js]: Good catch. I didn’t. It was a bad manual merge

**Comment by bfred-it** [comment]
Re: GHE

As long as PRs don’t break GHE we can still merge them. You’ll just miss this minor cleanup until your GHE is updated

**Comment by yakov116** [comment]
Maybe leave the move account sidebar for a bit let them at least have that

**Comment by bfred-it** [comment]
@yakov116 same answer applies.

**Pull request merged**